### DISCUSSION TITLE
Monitoring przejść granicznych

### DISCUSSION THREAD

**Discussion by sypianski** [openstreetmap-polska/ua-2022-map#76] [Ideas]
Prowadzę monitoring sytuacji i potrzeb na przejściach granicznych na [ukraina.grupagranica.pl/granica](ukraina.grupagranica.pl/granica). Znam QGIS, ale nie robiłem nigdy w OMS. Czy jest opcja, żeby naszą mapę podłączyć jakoś pod OMS i Wasz projekt?

**Comment by Mordechai23** [comment]
#73

**Comment by sypianski** [comment]
Odpowiedź tam dotyczy tylko jednej tabeli, która została tam uznana za niewiarygodną. My mamy własne dane.
